man/journald.conf.xml

```diff
@@ -348,7 +348,10 @@
         <literal>notice</literal> for <varname>MaxLevelKMsg=</varname>,
         <literal>info</literal> for <varname>MaxLevelConsole=</varname>,
         and <literal>emerg</literal> for
-        <varname>MaxLevelWall=</varname>.</para></listitem>
+        <varname>MaxLevelWall=</varname>. These settings may be
```

Comment:
Force update, and comment so people get a mail notification.

Yes, it fails, I think you missed a header or made a typo. You can click on the small red x to get to the logs.